2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -20,7 +20,7 @@
  {
   "input": "Of all the expressions we could dream up that give you some symmetric smooth graph with mass concentrated towards the middle, why is it that the theory of probability seems to have a special place in its heart for this particular expression?",
   "model": "nmt",
-  "translatedText": "De todas las expresiones que podríamos imaginar que dan una gráfica simétrica suave con masa concentrada hacia el centro, ¿por qué la teoría de la probabilidad parece tener un lugar especial en su corazón para esta expresión en particular?",
+  "translatedText": "De todas las expresiones que podríamos imaginar, que darían una gráfica simétrica, suave y con masa concentrada hacia el centro, ¿por qué parece que la teoría de la probabilidad tiene esta expresión en particular en un lugar especial de su corazón?",
```

I find the old sentence clearer and closer to the meaning of the English sentence. Maybe the "en particular" part is a bit redundant? I would also use "pareciera" instead of "parece", but that could be a regional preference:

```suggestion
  "translatedText": "De todas las expresiones que podríamos imaginar, que darían una gráfica simétrica, suave y con masa concentrada hacia el centro, ¿por qué la teoría de la probabilidad pareciera tener un lugar especial en su corazón para esta expresión?",
```